src/content/en/2020/markup.md

```diff
@@ -121,8 +120,6 @@ Here are the 10 most popular (normalized) languages in our sample. At first we c
 <figcaption>{{ figure_link(caption="Top 10 <code>lang</code> attribute values.", sheets_gid="2047285366", sql_file="pages_almanac_by_device_and_html_lang.sql") }}</figcaption>
 </figure>
 
-{# TODO(authors): Add an interpretation of the lang results. #}
```

Comment:
Explanation for removal: I’d argue there’s little to interpret, or to defer to methodology as anything we see here may have been introduced on that end.

Reply:
What conclusions would you like to see readers draw from this figure? If there's nothing to say about it, do we need it at all?

Reply:
This may raise a very good point here! If the data set suggests that this data may not be representative for the wider Web then we should probably take this out, because what is there to see?

Reply:
The dat set is crawled from a US crawler with `en-US` set as the preferred language. Not sure how common it is to redirect to a home page based on that locale?

Additionally, the set of URLs is based on the CrUX data based off of Chrome data - is that swayed towards western users (therefore explaining the low numbers of Asia sites other than Japanese)?

Therefore I think it raises a very good question as to whether we can rely on this data? At the very least we should add a caveat explaining these influences.

Reply:
I remember the point of this section was to help the readers better understand the data set they're looking at: e.g. where do most of the pages come from and what is the main language used/detected on them.

In my opinion, I think a pie chart with `en`, `en-us`, `ja`, `es` etc would do it here. I'd also add in the chart the `22.36%`  of all documents that specify no `lang` attribute.

Reply:
I think that's an understandable rational for inclusion, but don't think that's clear from the text and so should make it clear. How about adding some thing like "It should be noted that the HTTP Archive crawls from US data centres with English language settings, so looking at the language pages are written in will be skewed towards English. Nevertheless we present the `lang` attributes seen to give some context to the sites analysed." Or something like that.

I'd discourage pie charts (very difficult to measure the size of relative slices) and think a standard bar graph would better here:

![image](https://user-images.githubusercontent.com/10931297/101021949-981f6f80-3568-11eb-986b-a7f153bb22aa.png)

Reply:
I have nothing against using a bar chart here, especially that both desktop and mobile stats can be shown in this way.

Reply:
@bazzadp could you add that as a suggestion for @j9t to review/approve?

Reply:
Does look like I can do this but I wouild suggest something like this:

```

Here are the 10 most popular (normalized on case) languages in our sample. It is important to note that the HTTP Archive crawls from US data centres with English language settings, so looking at the language pages are written in will be skewed towards English. Nevertheless we present the lang attributes seen to give some context to the sites analysed.

{{ figure_markup(
  image="top-html-lang.png",
  alt="The top HTML lang attritbues.",
  caption="The top HTML `lang` attritutes.",
  description="Bar chart showing the top 10 `lang` attributes using in our crawl with 22.82% of desktop and 22.36% of mobile not setting this, `en` being used on 20.09% and 18.08% respectively, `ja` on 15.17% and 13.27%, `es` on 4.86% and 4.09% , `pt-br` on 2.65% and 2.84%, `ru` on 2.21% 2.53%, `en-gb` on 2.35% and 2.19%, `de` on 1.50% and 1.92%, and finally `fr` being used on 1.55% and 1.43% respectively",
  sheets_gid="2047285366",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQPKzFb574UnGTcfw5mcD1qR7RYHyGjQTc2hiMuYix0QoTH1DPe54Q2JucXL8bfZ6kjRoAfhk3ckudc/pubchart?oid=1873310240&format=interactive",
  width=600,
  height=371,
  sql_file="pages_almanac_by_device_and_html_lang.sql"
  )
}}
```

Reply:
Updated the page (with just minor edits).

Reply:
Suggestion to remove SGTM.

Reply:
Remove? Or keep the update? (We had updated this section.)

Reply:
I was referring to the removal of the table. Either way, LGTM.